repo/packages/E/elasticsearch/0/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
also need the entry for framework: true

example: https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/C/cassandra/0/package.json#L7